File: answers/jahnvit25/Conditionals/Q2.cpp
#include<stdio.h>
int main()
{
	int date,month,year,i;
    printf("Enter date");
    scanf("%d", &date);
    printf("Enter month");
    scanf("%d", &month);
    printf("Enter year");
    scanf("%d", &year);
    switch(month)
    {
    	case 1:
    	printf("january/%d/%d",date,year);
    	break;
    	case 2:
    	printf("february/%d/%d",date,year);
    	break;
    	case 3:
    	printf("march/%d/%d",date,year);
    	break;
    	case 4:
    	printf("april/%d/%d",date,year);
    	break;
    	case 5:
    	printf("may/%d/%d",date,year);
    	break;
    	case 6:
    	printf("june/%d/%d",date,year);
    	break;
    	case 7:
    	printf("july/%d/%d",date,year);
    	break;
    	case 8:
    	printf("august/%d/%d",date,year);
    	break;
    	case 9:
    	printf("september/%d/%d",date,year);
    	break;
    	case 10:
    	printf("october/%d/%d",date,year);
    	break;
    	case 11:
    	printf("november/%d/%d",date,year);
    	break;
    	case 12:
    	printf("december/%d/%d",date,year);
    	break;
    	default:
    		printf("wrong input");
	}
return(0);
}

File: answers/VrishtiSharma/Conditonals/Q2.py
def month(m):
    switcher = {
        1 : "January",
        2 : "February",
        3: "March",
        4 : "April",
        5 : "May",
        6 : "June",
        7 : "July",
        8 : "August",
        9 : "September",
        10 : "October",
        11 : "November",
        12 : "December"
    }
    return switcher.get (m,"Invalid Month")

d = int(input(" Enter the date : "))
m = int(input("Enter the month : "))
y = int(input("Enter the year : "))

print(month(m), d, y, end = " ")
